do_files_push.go

```diff
@@ -241,32 +245,58 @@ func doFilesPush(
 		response, err := client.UploadFile(project, request)
 
 		if err != nil {
-			return NewError(
-				hierr.Errorf(
-					err,
-					`unable to upload file "%s"`,
-					file,
-				),
+			if returnError(err) {
+				return NewError(
+					hierr.Errorf(
+						err,
+						`unable to upload file "%s"`,
+						file,
+					),
+
+					`Check, that you have enough permissions to upload file to`+
+						` the specified project`,
+				)
+			}
+			_, _ = os.Stderr.WriteString("Unable to upload file " + file + "\n")
+			failedFiles = append(failedFiles, file)
+		} else {
+			status := "new"
+			if response.Overwritten {
+				status = "overwritten"
+			}
 
-				`Check, that you have enough permissions to upload file to`+
-					` the specified project`,
+			fmt.Printf(
+				"%s (%s) %s [%d strings %d words]\n",
+				uri,
+				request.FileType,
+				status,
+				response.StringCount,
+				response.WordCount,
 			)
 		}
+	}
 
-		status := "new"
-		if response.Overwritten {
-			status = "overwritten"
-		}
+	if len(failedFiles) != 0 {
+		result = NewError(fmt.Errorf("failed to upload %d files", len(failedFiles)), "failed to upload files "+strings.Join(failedFiles, ", "))
+	}
 
-		fmt.Printf(
-			"%s (%s) %s [%d strings %d words]\n",
-			uri,
-			request.FileType,
-			status,
-			response.StringCount,
-			response.WordCount,
-		)
+	return result
+}
+
+func returnError(err error) bool {
+	if err.Error() == "failed to upload original file: unable to authenticate: "+
```

Comment:
Pls format code around `+` sign (here and other files)

Reply:
Code formatted by gofmt

Reply:
ok